bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/Configuration.java

```diff
@@ -221,7 +221,7 @@ public String toString() {
                 sb.append(", ");
             }
             sb.append(String.format("{key=%s; type=%s; value=%s}", prop.getKey(),
-                    prop.getValue().getClass().getSimpleName(), prop.getValue()));
+                    prop.getValue() != null ? prop.getValue().getClass().getSimpleName() : "?", prop.getValue()));
```

Comment:
I prefer to use a temp. variable instead of calling the method `getValue()` three times if only one call is necessary. :wink: